dogen/plugins/dist_git.py

```diff
@@ -14,6 +14,7 @@ def info():
     def inject_args(parser):
         parser.add_argument('--dist-git-enable', action='store_true', help='Enables dist-git plugin')
         parser.add_argument('--dist-git-assume-yes', action='store_true', help='Skip interactive mode and answer all question with "yes"')
+        parser.add_argument('--scratch', action='store_true', help='Scratch build')
```

Comment:
Probably should be: `--dist-git-scratch`. The reason is that the `--dist-git` string is a prefix for all arguments for the dist-git plugin.